### DISCUSSION TITLE
Weird Display after update auf Release: v14.0.0-RC8

### DISCUSSION THREAD

**Discussion by ujagusch** [jomjol/AI-on-the-edge-device#1960] [General] [answered]
Nach dem update auf Release: v14.0.0-RC8 zeigt das Display in Recognition und Overview folgende Anzeige:

![image](http://taxi-troisdorf.de/bilder/fehler2.jpg)]

Die Auswertung der Digital und Analoganzeigen ist aber korrekt, ebenso werden diese in Recognition korrekt angezeigt.
Weder erneutes Betätigen von Overview oder Recognition ändert etwas an der Anzeige.

Dieser Fehler tritt erst nach dem Update auf Version v14.0.0-RC3 auf.

UJagusch

**Comment by Slider0007** [comment]
@ujagusch: Das ist seltsam. Kannst du bitte mit F12 die Entwicklerkonsole in deinem Webbrowser (z.B. Chrome, Edge, ...) öffnen, auf den Reiter Netzwerk wechseln, die Seite neu laden oder den Overview bzw. Recognisition Button betätigen und einen Screenshot einstellen (mich interessiert die Größe des alg_roi.jpg files).

Ausgabe vgl. dieser:
![image](https://user-images.githubusercontent.com/115730895/216105392-67f13ede-fe4a-4f30-94e3-b3f1882dd8fa.png)

**Reply by ujagusch** [reply]
So hier der screenshot

![image](http://taxi-troisdorf.de/bilder/fehler3.jpg)

Größe ist immer 137k, erscheint mir etwas groß?

Ujagusch

**Comment by Slider0007** [comment]
@ujagusch: Danke für das schnelle Feedback. Die Größe ist abhängig von der Anzahl der ROIs, die über das eigentliche Bild gelegt werden. Es scheint, dass wir hier zu grenzwertig unterwegs sind. Aktuell ist die Größe auf 128kB limitiert. Kannst du bitte folgende modifizierte RC8 Version testen und Rückmeldung geben:  https://github.com/jomjol/AI-on-the-edge-device/actions/runs/4067865479

Einfach das update file über OTA einspielen:
![image](https://user-images.githubusercontent.com/115730895/216141960-cb04c505-446f-4c5a-b57b-39355c341666.png)

**Comment by ujagusch** [comment]
OK - sobald ich wieder zuhause bin probier ich das aus.

Slider0007 ***@***.***> schrieb am Mi., 1. Feb. 2023, 20:22:

**Comment by Slider0007** [comment, marked as answer]
Ich habe ein Ticket zum Problem geöffnet. Bitte hier rückmelden --> #1961. Danke!
